src/allencell_ml_segmenter/main/main_widget.py

```diff
@@ -185,6 +196,7 @@ def _disable_non_prediction_tabs(self, _: Event) -> None:
         """
         self._window_container.setTabEnabled(0, False)
         self._window_container.setTabEnabled(1, False)
+        self._window_container.setTabEnabled(3, True)  # TODO remove
```

Comment:
why is the same statement on lines 199 and 188 with `TODO remove`?  One needs to be cleaned up?

Reply:
I'll remove this in a subsequent PR before merging this to main. For now it makes thresholding easier to test (the tab is instantly available)

The other line is the behavior we want, but this line makes it so that I can jump to the thresholding tab